client/plots/matrix.interactivity.js

```diff
@@ -256,6 +257,48 @@ export function setInteractivity(self) {
 					self.dom.clickMenu.d.selectAll('*').remove()
 				})
 		}
+
+		const showDZImages = JSON.parse(sessionStorage.getItem('optionalFeatures') || `{}`)?.showDZImages
+		if (q.DZImages && showDZImages) {
+			async function fetchHnEImages() {
+				return dofetch3('sampledzimages', {
```

Comment:
why not just `const images= await dofetch...`

Reply:
@aacic the `async` keyword should be used in the function declaration above, `self.mouseclick = function (event, data) {`, and not be declaring an async function inside this function, so you can then simply do `const images = awaif dofetch3 (...)`

Reply:
Refactored.